src/assets/styles/variables.scss

```diff
@@ -6,6 +6,7 @@ $medium-gray: #99999a;
 $light-gray: #d4d4d4;
 $lighter-gray: #ededed;
 $lightest-gray: #f9f9f9;
+$white: #ffffff;
```

Comment:
@Rajmohanrk This is not needed. Anywhere we use the white color, we just define it as  `color: white`. This is simple enough we don't need an extra variable just to call `color: $white`